.github/workflows/docker/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# This docker compose file is used to create a backend environment for the e2e.yml workflow.
```

Comment:
I don't love putting additional files in the .github/workflows folder or the .github tree at all. This should probably be in the e2e folder itself as a support file.

Reply:
Moved it to the `e2e/support` directory.